lib/rufus/lua/lib.rb

```diff
@@ -49,24 +78,20 @@ def self.path
 
       f ? f.name : nil
     end
-
     #
     # attach functions
 
     attach_function :lua_close, [ :pointer ], :void
 
-    attach_function :luaL_openlibs, [ :pointer ], :void
-
-    attach_function :lua_call, [ :pointer, :int, :int ], :void
     %w[ base package string table math io os debug ].each do |libname|
-      attach_function "luaopen_#{libname}", [ :pointer ], :void
+      attach_function "luaopen_#{libname}", [ :pointer ], :int
```

Comment:
This was just wrong, this method returns an int in both Lua 5.1 and Lua 5.3:

https://www.lua.org/source/5.1/lmathlib.c.html#luaopen_math
https://www.lua.org/source/5.3/lmathlib.c.html#luaopen_math